specification/contosowidgetmanager/Contoso.WidgetManager.Shared/main.tsp

```diff
@@ -4,5 +4,5 @@ model FakedSharedModel {
   tag: string;
 
   @doc("The created date.")
-  createdDate: zonedDateTime;
-}
\ No newline at end of file
+  createdDate: utcDateTime;
```

Comment:
```suggestion
  createdAt: utcDateTime;
```

To follow our naming guidelines: https://github.com/microsoft/api-guidelines/blob/vNext/azure/ConsiderationsForServiceDesign.md#common-names

Reply:
@weshaggard: Do you think this property name should be updated in the sample per @JeffreyRichter suggestion?  If so, I will update this in a separate PR.

Reply:
Yes I'd like to keep the sample matching the guidelines.